### DISCUSSION TITLE
fix bad problem (cloudflare)

### DISCUSSION THREAD

**Discussion by Sepera-okeq** [Zero6992/chatGPT-discord-bot#85] [General]
Here the guys were able to fix this problem, maybe you can take a look at how it is fixed? https://github.com/Cubicpath/ChatGPT-GUI

![image](https://user-images.githubusercontent.com/54331959/208303056-bc8dd97b-c38b-447c-85b0-328f0c81459e.png)

**Comment by Zero6992** [comment]
Will check, thanks
